content/_index.md

```diff
@@ -25,16 +24,16 @@ carousel:
             Define infrastructure in JavaScript, TypeScript, Python, Go, or
             any .NET language, including C#, F#, and VB.
         - title: Your favorite tools.
-          description: Use familiar IDEs, test frameworks, and tools.
+          description: Use familiar IDEs, test frameworks, and tools. Resource mocking for easy verification.
```

Comment:
```suggestion
          description: Use familiar IDEs, test frameworks, and tools.
```

We already say test frameworks -- doesn't feel like we need to complicate the point with the mocking sentence.

Reply:
I'm okay with this edit.